packages/oid4vci-holder/src/agent/OID4VCIHolder.ts

```diff
@@ -871,8 +871,16 @@ export class OID4VCIHolder implements IAgentPlugin {
       logger.log(`Will not persist credential, since we are signing as a holder and the issuer asked not to persist`)
     } else {
       logger.log(`Persisting credential`, persistCredential)
-      // @ts-ignore
-      const vcHash = await context.agent.dataStoreSaveVerifiableCredential({ verifiableCredential: persistCredential })
+      const vcHash = await context.agent.crsAddCredential({
+        credential: {
+          rawDocument: JSON.stringify(persistCredential),
+          credentialRole: CredentialRole.HOLDER,
+          issuerCorrelationType: CredentialCorrelationType.DID,
+          issuerCorrelationId: (persistCredential as any).issuer, // FIXME BEFORE PR
```

Comment:
FIXMEs mentioned in the test

Reply:
fixed
(The FIXME's were supposed to be in the draft PR to help address them)